### PR TITLE
Feat: Uses `ProductCard` component from FSUI

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

This PR is part of the Component Placement project. Replaces `Card` for `ProductCard`

## How does it work?

- Uses `ProductCard` component from FSUI instead of `Card`
- Renames  `ProductCard` prop `actionabled` to `actionable`

## How to test it?

- Nothing should change visually on the component.
- Check on the [preview](https://preview-34--gatsby.preview.vtex.app) if the `ProductCard` is working fine as [before](https://gatsby.vtex.app/). 

- The `ProductCard` component appear on the **[Homepage](https://preview-34--gatsby.preview.vtex.app)** and **[PLP](https://preview-34--gatsby.preview.vtex.app/office)**:

|Homepage - ProductShelf|
|-|
image

|Homepage|
|-|
image

|PLP - Product Listing Page|
|-|
plp

## Checklist

<em>You may erase this after checking them all ;)</em>
- [x] After https://github.com/vtex/faststore/pull/1272 is merged, update faststore/ui version on stores and merge those PRs as well
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#12](https://github.com/vtex-sites/gatsby.store/pull/12))*

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3475,6 +3475,16 @@
     tiny-warning "^1.0.3"
     tslib "^2.3.0"
 
+"@reach/router@^1.3.4":
```
Comment (filipewl) [inline yarn.lock:3478]: Did you happen to commit this file by mistake by any chance?

I saw you had modified `package.json` [before](https://github.com/vtex-sites/gatsby.store/pull/34/commits/37aed14e446f71c9c3ae959886bb887d23349184) but it's not here now so I'm not sure this file is needed.

**Review by filipewl — APPROVED** [review]
LGTM!

Please take a look at my other comment.

File: yarn.lock (hunk 1 above)
Comment (hellofanny) [reply yarn.lock:3478]: Oh yeah! My bad! I'll remove it! Thanks!

**Review by renatamottam — APPROVED** [review]
Nice 👏

**Pull request merged**